File: Programmers/Lv.0/수 조작하기 1/181926.js
function solution(n, control) {
    let answer = n;
    
    for (const s of [...control]) {
        if (s === 'w') {
            answer += 1;
        } else if (s === 's') {
            answer -= 1;
        } else if (s === 'd') {
            answer += 10;
        } else {
            answer -= 10;
        }
    }
    
    return answer;
}

File: Programmers/Lv.0/수 조작하기 1/181926.java
class Solution {
    public int solution(int n, String control) {
        int answer = n;
        
        for (String s : control.split("")) {
            if ("w".equals(s)) {
                answer += 1;
            } else if ("s".equals(s)) {
                answer -= 1;
            } else if ("d".equals(s)) {
                answer += 10;
            } else {
                answer -= 10;
            }
        }
        
        return answer;
    }
}
